README.md

```diff
@@ -56,37 +57,30 @@ Browse the [HASH roadmap] for more information about upcoming features and relea
 
 This repository's contents is divided across four primary sections:
 
-- [`/apps` - applications](#applications): the primary code behind our runnable applications
-- [`/blocks` - blocks](#blocks): our public [Block Protocol] blocks
-- [`/infra`- infrastructure](#infrastructure): deployment scripts and other tools used to run our apps
-- [`/libs` - libraries](#libraries): includes npm packages and Rust crates
+- [**`/apps`**](/apps) contains the primary code powering our runnable [applications](#applications)
+- [**`/blocks`**](/blocks) contains our public _Block Protocol_ [blocks](#blocks)
+- [**`/infra`**](/infra) houses deployment scripts, utilities and other [infrastructure](#infrastructure) useful in running our apps
+- [**`/libs`**](/libs) contains all of our public [libraries](#libraries), including npm packages and Rust crates
```

Comment:
Rephrase to cover two points:
1. most of the stuff in `libs` aren't 'public' in the sense that they aren't available via a registry, they're mostly just used within this repo
2. we have more libraries in `blockprotocol` so this isn't all of ours
```suggestion
- [**`/libs`**](/libs) contains [libraries](#libraries) including npm packages and Rust crates
```